app/packages/core/src/components/Filters/NumericFieldFilter.tsx

```diff
@@ -157,6 +164,12 @@ const NumericFieldFilter = ({
     (nonfinites.length === 1 && nonfinites[0][0] === "none")
   );
 
+  const hasNone =
+    nonfinites.some((x) => x[0] === "none") &&
+    nonfinites.find((x) => x[0] === "none")![1].count !== 0;
+  const isSliderAtInitialPostion =
+    bounds[0] === values[0] && bounds[1] === values[1];
```

Comment:
`isSliderAtInitialPosition` and `hasDefaultRange` should mean the same thing. Maybe you can fix the `isDefaultRange` selector?

Reply:
I will fix `hasDefaultRange` and name is `isDefaultRange`. I wasn't sure about `hasDefaultRange` so didn't touch it.